pkg/bpf/bpf_restart.go

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2023 The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package bpf
+
+import (
+	"context"
+	"os"
+	"syscall"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kmesh.net/kmesh/pkg/utils"
+)
+
+const (
+	NewStart = iota
+	Restart
+	Update
+	Reload
```

Comment:
These constants also defined in `pkg/version`, can they be defined in one place?

Reply:
moved